eng/pipelines/runtime.yml

```diff
@@ -246,7 +246,7 @@ extends:
             extraStepsTemplate: /eng/pipelines/coreclr/nativeaot-post-build-steps.yml
             extraStepsParameters:
               creator: dotnet-bot
-              testBuildArgs: nativeaot tree nativeaot /p:BuildNativeAotFrameworkObjects=true
+              testBuildArgs: 'nativeaot tree ";nativeaot;Loader;Interop;" /p:BuildNativeAotFrameworkObjects=true'
```

Comment:
@dotnet/runtime-infrastructure are we using the semicolon-delimited list as an argument to test build `tree` somewhere?

I was having trouble getting it to work - cmd.exe considers semicolons the same as space when it comes to %1 %2 argument processing (today I learned...) so it has to be quoted. But then nobody removed the quote and MSBuild was looking for directory `"nativeat`. Adding an extra empty entry like I'm doing here hacks around it but I wonder if we should find a different list separator.